cedar-policy-validator/src/schema.rs

```diff
@@ -621,7 +621,8 @@ impl ValidatorNamespaceDef {
         namespace: Option<&Name>,
     ) -> Result<EntityUID> {
         let namespaced_action_type = if let Some(action_ty) = &action_id.ty {
-            Name::from_normalized_str(action_ty).map_err(SchemaError::EntityTypeParseError)?
+            Self::parse_possibly_qualified_name_with_default_namespace(action_ty, namespace)
+                .map_err(SchemaError::NamespaceParseError)?
```

Comment:
Error type changed from `EntityTypeParseError` to `NamespaceParseError`, was that intentional?

Reply:
Good catch. That does not look right.

Reply:
Sure, I wasn't sure which one we wanted, since we are parsing a namespace as well?